geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/SimpleRasterIngest.scala

```diff
@@ -31,16 +31,18 @@ import org.joda.time.{DateTime, DateTimeZone}
 import org.locationtech.geomesa.core.index.DecodedIndex
 import org.locationtech.geomesa.raster.data.{Raster, AccumuloCoverageStore}
 import org.locationtech.geomesa.raster.util.RasterUtils.IngestRasterParams
+import org.locationtech.geomesa.tools.Utils.Formats._
 import org.locationtech.geomesa.utils.geohash.BoundingBox
 
+import scala.collection.parallel.ForkJoinTaskSupport
 import scala.util.Try
 
 object SimpleRasterIngest {
 
   def getReader(imageFile: File, imageType: String): AbstractGridCoverage2DReader = {
     imageType match {
-      case "tiff" => getTiffReader(imageFile)
-      case "dted" => getDtedReader(imageFile)
+      case TIFF => getTiffReader(imageFile)
```

Comment:
Here is principle I used for file type:
- One type for one kind of images to simplify file type related process later on in the codes.
- To ingest Geotiff files under a directory, use format "-f tiff", it supports file extensions .tif and .tiff.
- To ingest DTED files under a directory, use format "-f dted", it supports file extensions .dt0. .dt1 and .dt2.